src/makeCard.js

```diff
@@ -12,6 +12,36 @@ const makeCard = (show) => {
     </div>
     <input class="comment button" type="button" value="Comment">
   `;
+  const popup = document.querySelector('.createPopup');
+  showCard.innerHTML = `
+    <img src="${show.image.medium}">
+    ${show.name}
+    <input type="button" value="Comment" class="comment"> 
+  `;
+
+  showCard.querySelector('.comment').addEventListener('click', () => {
+    popup.classList.add('invisible');
+    popup.innerHTML = `
+    <div class="popupContent">
+      <div class="title-close">
+        <h1 class="showTitle">${show.name}</h1>
+        <button type="button" class="closePopup">X</button>
+      </div>
+      <img src="${show.image.medium}">
+      <p>${show.summary}</p>
+      <div class="form">
+        <input type="text" id="name" placeholder="Your name">
+        <textarea id="comments" placeholder="Type your opinion here..."></textarea>
+        <button class="submitOpinion" type="submit">Submit</button>
+      </div>
+    </div>
+   `;
```

Comment:
[Optional] You could put this as a separate function (with the parameter (show), maybe) in another file, and import it here. It is fine now, but later, it might make testing easier.